daisy_workflows/image_test/metadata-script/metadata-script-integrity.subwf.json

```diff
@@ -12,8 +14,10 @@
             "instance_tag": "integrity",
             "startup-script-url": "https://raw.githubusercontent.com/GoogleCloudPlatform/compute-image-tools/master/daisy_workflows/image_test/metadata-script/metadata-script-test-startup-hash.sh",
             "shutdown-script-url": "https://raw.githubusercontent.com/GoogleCloudPlatform/compute-image-tools/master/daisy_workflows/image_test/metadata-script/metadata-script-test-shutdown-hash.sh",
-            "startup_msg": "631efd12afdf1460ad5f236fe272dccb",
-            "shutdown_msg": "cfcddad49687cb301ffb029a5b418866"
+            "windows-startup-script-ps1": "https://raw.githubusercontent.com/GoogleCloudPlatform/compute-image-tools/master/daisy_workflows/image_test/metadata-script/metadata-script-test-startup-hash.ps1",
```

Comment:
Don't rely on the connection with github, also this makes updates difficult to test. Instead add the startup script to Sources for this workflow

Reply:
I agree with you, partly.
I had to do this because I want to test that the agent is capable of reading scripts by downloading them from https. I used github as a kind of trustful source. Do you suggest something else? I could serve a file on another instance and test if it downloads correctly by booting yet another instance, if you see it as good idea.

Reply:
We need to test URL download (github isn't too bad of a place for this), GCS download (both authenticated and unauthenticated), and running directly from metadata. We can leave this as is for now though.